sonaranalyzer-dotnet/src/Tests/SonarAnalyzer.UnitTest/TestCases/MutableFieldsShouldNotBePublicStatic.cs

```diff
@@ -47,5 +52,14 @@ class InvalidCases
         public static SortedList<string, string> sortedListString; // Noncompliant
         public static ObservableCollection<string> observableCollectionString; // Noncompliant
         public static Foo foo; // Noncompliant
+

```

Comment:
These are a bit different, as the type is immutable, but the class field is not (e.g. List x = new ImmutableList()). It would be good to add an explicit one (e.g. ImmutableList x = new ImmutableList()).